### DISCUSSION TITLE
Add a guesttimate of the hours to complete each given track/module

### DISCUSSION THREAD

**Discussion by mattstauffer** [tightenco/onramp#449] [Ideas]
(Thanks Chris Sciora for the suggestion)

Chris' guess is 15-30 hours per module.

Not sure how we would actually figure this out. Maybe add a "average time to consume" to each resource? That'd be an imperfect start.

**Comment by Gummibeer** [comment]
I think that maintaining that value manually isn't as fancy as it could be - but the simplest and most flexible solution. The alternative would be to get the video duration of youtube resources and text length of articles. Calculate the read and watch time and multiply by any modifier to include time for practice and so on.
